src/select/Menu.js

```diff
@@ -114,33 +118,40 @@ class Menu extends Component {
       multiStaticOptions.push(staticOption)
     }
 
-
     return (
-      <div className="crane-select-menu" ref={menuRef} role="listbox">
+      <div
+        aria-label="menu"
```

Comment:
why the `aria-label` addition?

Reply:
I don't think it'll hurt anything...though having static aria-labels makes me nervous.  In this case, it would only ever be an issue if you had two react-crane menus open at the same time and I don't think that's likely.

Reply:
`Axe Linter (aria-input-field-name): Ensures every ARIA input field has an accessible name`

The Axe Linter caught this one!  I thought about making it dynamic, but I didn't want to use the same dynamic aria-label passed for other references, and this is a main tag, so I'm not sure how much it would differ anyways.